src/endpoint/sts/ops/sts_post_assume_role.js

```diff
@@ -12,6 +12,9 @@ const { CONTENT_TYPE_APP_FORM_URLENCODED } = require('../../../util/http_utils')
  */
 async function assume_role(req) {
     dbg.log1('sts_post_assume_role body: ', req.body);
+    const ms_until_expiration = _parse_sts_duration(req.body.duration_seconds);
+    const seconds_until_expiration = Math.ceil(ms_until_expiration / 1000);
+    const expiration = _calculate_expiration_time(ms_until_expiration);
```

Comment:
Lets keep the same terms
```suggestion
    const duration_ms = _parse_sts_duration(req.body.duration_seconds);
    const duration_sec = Math.ceil(duration_ms / 1000);
    const expiration_time = Date.now() + duration_ms;
```